src/sentry/snuba/discover.py

```diff
@@ -209,6 +209,7 @@ def query(
     referrer=None,
     auto_fields=False,
     auto_aggregations=False,
+    auto_equation_fields=False,
```

Comment:
can we rename this to something more descriptive like "include_equation_fields" or "add_equation_fields" or something?

Reply:
I like `include_equation_fields` over `add_equation_fields`, `include` makes me feel more like something is happening for me and reminds me of include filters for URL endpoints. `add` feels different